addon/libconiks.js

```diff
@@ -0,0 +1 @@
+../chrome/content/libconiks.js
```

Comment:
Oops :( 

> But maybe we want to put this file in addon/coniks/libconiks.js, in which case it'll need an additional ../

Yup, I'm not sure should we put this file into `coniks` directory or not, but it's obvious now.